bin/oneoff/populate_metrics_from_tsv

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env ruby
+
+# This script reads a TSV of (INTEGER) data and writes the data to the metrics
+# table.
+
+require_relative '../../dashboard/config/environment'
+require 'csv'
+
+# Prompts the user for the TSV file and reads it. The file is expected to have
+# two columns, a column of dates (metric_on) and a column of values, as well as
+# a header row.
+# @return data [Array]
+def get_data
+  print 'TSV FILE: '
+  tsv_file = gets.chomp
+
+  data = []
+  CSV.foreach(tsv_file, headers: true, col_sep: "\t") do |row|
+    data << row
+    puts row
```

Comment:
Noisy enough that I expect to delete it in a future PR. But at least for now, I want the output.